config/prod/lheath-t3.yaml

```diff
@@ -0,0 +1,47 @@
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: lheath-ec2-t3
```

Comment:
you may want to change your instance type later on so maybe just drop `t3` from the name?